site/content/docs/browser-checks/_index.md

```diff
@@ -9,22 +9,43 @@ aliases:
     - /docs/browser-checks/quickstart/
 ---
 
+This quick start gives you all the info to create your first Browser check with Checkly. You should have some prior
+knowledge of working with Javascript and/or Node.js.
 
+## What is a Browser check?
 
-This quick start should give you all the info to create your first browser check with Checkly. You should have some prior
-knowledge working with Javascript and/or Node.js.
+A Browser check is a Node.js script that controls a Chrome browser to mimic user behavior.
+Load a web page, click a link, fill a form input – do everything your visitors might do and check if these interactions lead to the correct results.
 
-## What is a browser check?
+Your critical interactions might be:
 
-The five lines of code below are already a valid browser check. Its usefulness might be not that great, but still...
+- That the shopping cart is visible.
```

Comment:
The first point doesn't work. "That the shopping cart is visible" is not an interaction. Verifying that an element appears is more of an assertion. I would substitute this with a third interaction. We can add a one liner below explaining how assertions factor into the equation (automated interactions + assertions = confidence things are working as expected).

Reply:
👍